tsconfig.json

```diff
@@ -10,7 +10,7 @@
     "skipLibCheck": true,
     "esModuleInterop": true,
     "allowSyntheticDefaultImports": true,
-    "strict": true,
+    "strict": false,
```

Comment:
Why turn off strict mode?

Reply:
Thanks for discovering this... This is something I put to remove some alerts in my IDE but just for developing. I forgot to change it back to true